app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java

```diff
@@ -209,6 +209,7 @@ public void next() {
     void setCurrentTitleAndDescriptions(Title title, List<Description> descriptions) {
         setCurrentUploadTitle(title);
         setCurrentUploadDescriptions(descriptions);
+        store.putString("Title", title.toString());
```

Comment:
Is this change related?

Reply:
no , this change of another issue , i didn't know that every pull request needs a new branch , i'll try to fix it.